02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,129 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>All About Brady</title>
+  <meta charset="utf-8">
 </head>
-<body>
+
+<body style="background-color:lightblue">
+  <h1 style="text-align:center">
+    MoBrady MoProblems
+  </h1>
+  <p style="text-align:center">
+    <em>You can sleep when you're dead</em>
+  </p>
+    <div style="text-align:center">
```

Comment:
Indentation: After closing `p` tag the following opening tag should start on the same level